src/Command/InfectionCommand.php

```diff
@@ -481,38 +487,68 @@ private function getTestFrameworkExtraOptions(string $testFrameworkKey): TestFra
     protected function initialize(InputInterface $input, OutputInterface $output)
     {
         parent::initialize($input, $output);
+
         $locator = $this->getContainer()->get('locator');
+
         if ($customConfigPath = $input->getOption('configuration')) {
             $locator->locate($customConfigPath);
-            $this->doInitialize($input);
+        } else {
+            $this->runConfigurationCommand($locator);
+        }
 
-            return;
+        $this->io = $this->getApplication()->getIO();
+        $this->skipCoverage = \strlen(trim($input->getOption('coverage'))) > 0;
+        $this->eventDispatcher = $this->getContainer()->get('dispatcher');
+    }
+
+    public function isDebuggerOrAdditionalParametersExist(): bool
```

Comment:
not sure about name of this function :)

Reply:
What about simply `hasDebugger`

Reply:
nope. because this method is not only about debugger